experimental/generated/beats/fields.ecs.yml

```diff
@@ -7162,6 +7208,59 @@
       ignore_above: 1024
       description: All the user names or other user identifiers seen on the event.
       default_field: false
+  - name: risk
```

Comment:
What entity is this set associated with? I didn't see them in the `fields.csv` file!?